proposals/mail-helper-refactor.md

```diff
@@ -228,7 +228,7 @@ msg.attachment = [
 msg.template_id = TemplateId('13b8f94f-bcae-4ec6-b752-70d6cb59f932')
 
 msg.global_header = Header('X-Day', 'Monday')
-msg.global_headers = [
+msg.global_header = [
```

Comment:
This can be multiple headers, should it stay plural?

Reply:
I think this should remain plural.